src/lib/accessibility/Behaviors/Button/buttonBehavior.ts

```diff
@@ -3,14 +3,14 @@ import { Accessibility } from '../../interfaces'
 /**
  * @description
  *  Adds role='button' if element type is other than 'button'. This allows screen readers to handle the component as a button.
- *  Adds attribute 'aria-disabled=true' based on the property 'disabled'.
+ *  Adds attribute 'aria-disabled=true' based on the property 'disabled'. This can be overriden by directly providing 'aria-disabled' property to the component.
```

Comment:
while do understand that it could be a rather subjective matter, at the same it seems that 
> This can be overriden by providing 'aria-disabled' property directly to the component.

sounds more natural